### DISCUSSION TITLE
能增加漫画资源网站吗？ 百漫屋 千漫谷 85漫画网

### DISCUSSION THREAD

**Discussion by OuruolaW** [kanasimi/work_crawler#581] [General]
www.ql1.net    漫画挺全的，我看过问题区了，没有这个网站
https://m.85iu.com/manhua/woyumojunbukeshuo/     这个网站很奇怪首页进不去，但可以进漫画页，搜索也能正常进行
https://www.wuforcongress.com/ 速度比较慢

**Comment by Z-fly** [comment]
这个不是想加立刻就能加的，要编写对应的代码规则

**Reply by OuruolaW** [reply]
明白，非常感谢
